js/colourpicker.js

```diff
@@ -1145,7 +1115,7 @@
     function getLastColor(input) {
       if (input.data('colourpicker-lastChange')) {
         return input.data('colourpicker-lastChange');
-      } else if (input.parent().is('.palette-limited')) {
+      } else if (input.parents('.colourpicker').is('.palette-limited')) {
```

Comment:
Oooh! Yes! I didn't know about closest. My jQuery skills basically come down to what comes up when I google. =)